test/e2e/run_test.go

```diff
@@ -267,13 +282,17 @@ var _ = Describe("Podman run", func() {
 		Expect(stdoutLines).Should(HaveLen(1))
 		Expect(stdoutLines[0]).Should(Equal(uniqueString))
 
-		SkipIfRemote("External overlay only work locally")
-		if os.Getenv("container") != "" {
-			Skip("Overlay mounts not supported when running in a container")
+		// The rest of these tests only work locally and not containerized
+		if IsRemote() || os.Getenv("container") != "" {
+			GinkgoWriter.Println("Bypassing subsequent tests due to remote or container environment")
+			return
 		}
+		// And when rootless, we need fuse-overlayfs
+		// FIXME FIXME: is this still the case in 2023?
```

Comment:
Special attention here please

Reply:
that depends on the kernel, but I think it is safe at this point to assume native overlay works for rootless